package.json

```diff
@@ -39,10 +40,23 @@
     "neutrino-preset-react": "4.2.3",
     "ngreact": "0.4.1",
     "numeral": "2.0.6",
+    "popper.js": "^1.12.5",
     "raw-loader": "0.5.1",
-    "react": "15.3.2",
-    "react-dom": "15.3.2",
-    "taskcluster-client": "2.5.0"
+    "react": "^15.4.2",
```

Comment:
Don't forget to remove these react changes and other package updates from this PR :-)